pkg/hooks/hook_env.go

```diff
@@ -46,6 +46,10 @@ func (h *HookEnv) Event() (HookType, error) {
 	return AsHookType(os.Getenv("HOOK_EVENT"))
 }
 
+func (h *HookEnv) EnvPath() string {
+	return os.Getenv("HOOKED_ENV_PATH")
```

Comment:
although we probably shouldn't change those because it'd be hard to preserve compatibility with hooks